src/Microsoft.IdentityModel.Logging/IdentityModelTelemetryUtil.cs

```diff
@@ -44,7 +44,7 @@ public static class IdentityModelTelemetryUtil
         /// <summary>
         /// Get the string that represents the client version.
         /// </summary>
-        public static string ClientVer => typeof(IdentityModelTelemetryUtil).GetTypeInfo().Assembly.GetName().Version.ToString();
+        public static string ClientVer => typeof(IdentityModelTelemetryUtil).GetTypeInfo().Assembly!.GetName().Version!.ToString();
```

Comment:
same comment, why is this required?

Reply:
IDE throws error code CS8602-Dereference of a possibly null reference. Version is not nullable in System.Reflection so it causes noise

Reply:
should be `GetTypeInfo().Assembly.GetName().Version?.ToString();` instead

Reply:
Changed implementation, pls review again for correctness. Thanks!

Reply:
I would declare this as null: `public static string? ClientVer => ...` and then earlier handle the null by passing an empty string. We wouldn't want to mess up with the values sent to eSTS. Maybe something like `[versionTelemetry] = ClientVer ?? string.Empy`